docs/model_developers/handlers.rst

```diff
@@ -125,6 +125,8 @@ For example:
         # ...
         ctx.debug("Creating loadbalancer with security group %(sg)s", sg=sg_id)
 
+In addition, the :class:`~inmanta.agent.handler.KwargsLogger` provides similar functionalities but without being
+restricted to an Agent context.
```

Comment:
This feel very lightweight as far as documentation goes, but I don't quite see how to make it better. The changelog message could also use some more care.

Reply:
I agree with the changelog message, I'll add a comment there. As to this being lightweight, yes but I also agree there's not much more to say. One thing I would make sure to mention is the interface. e.g.
```suggestion
This logger implements the `~inmanta.agent.handler.IBasicLogging` logging interface. An alternative implementation that just logs to the Python logger is provided in `~inmanta.agent.handler.KwargsLogger`. This logger is not meant to be used in actual handlers but it can be used for the automated testing of helper methods that accept a `IBasicLogging` instance. In production, these helpers would receive the actual `HandlerContext` and log appropriately, while for testing the `KwargsLogger` can be passed.
```

Just a quick draft, I'll leave it up to you to review and/or cherry-pick what you think appropriate.